repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/classloader/ClassLoaderTest.groovy

```diff
@@ -104,7 +95,6 @@ class ClassLoaderTest extends ReposeValveTest {
         then: "the request header should equal BAR"
         mc.handlings.size() == 1
         mc.receivedResponse.code == "200"
-        mc.handlings[0].request.headers.getFirstValue("FOO") == "BAR"
```

Comment:
Gotcha. Indeed, only one method was overridden... which assumes as a certain interaction with the wrapper.

I suppose you could replace that wrapper with our wrapper to get behavior you could test in Deproxy. Since you tied these projects into the rest of our build, you should be able to now. But it probably doesn't matter that much, since the next filter would throw an Exception if things weren't as expected. So fair enough.